src/actions/jira/jira.ts

```diff
@@ -1,133 +1,212 @@
+          const projects = await client.getProjects()
+          const projectOptions: { name: string, label: string }[] = projects.map((p: any) => {
+            return { name: p.id, label: p.name }
+          })
 
-    const form = new Hub.ActionForm()
-    try {
-      const jira = this.jiraClientFromRequest(request)
-
-      const [projects, issueTypes] = await Promise.all([
-        jira.listProjects(),
-        jira.listIssueTypes(),
-      ])
-
-      form.fields = [{
-        default: projects[0].id,
-        label: "Project",
-        name: "project",
-        options: projects.map((p: any) => {
-          return {name: p.id, label: p.name}
-        }),
-        type: "select",
-        required: true,
-      }, {
-        label: "Summary",
-        name: "summary",
-        type: "string",
-        required: true,
-      }, {
-        label: "Description",
-        name: "description",
-        type: "textarea",
-        required: true,
-      }, {
-        default: issueTypes[0].id,
-        label: "Issue Type",
-        name: "issueType",
-        type: "select",
-        options: issueTypes
-          .filter((i: any) => i.description)
-          .map((p: any) => {
-            return {name: p.id, label: p.name}
-          }),
-        required: true,
-      }]
-    } catch (e) {
-      form.error = e
+          const issueTypesOptions = [{
+            name: "10000",
+            label: "Epic",
+          }, {
+            name: "10001",
+            label: "Story",
+          }, {
+            name: "10002",
+            label: "Task",
+          }, {
+            name: "10003",
+            label: "Sub-task",
+          }, {
+            name: "10004",
+            label: "Bug",
+          }]          
+          projectOptions.sort((a, b) => ((a.label < b.label) ? -1 : 1 ))
+          issueTypesOptions.sort((a, b) => ((a.name > b.name) ? -1 : 1 ))
+
+          const form = new Hub.ActionForm()
+          form.fields = [{
+            default: projectOptions[0].name,
+            label: "Project",
+            name: "project",
+            options: projectOptions,
+            type: "select",
+            required: true,
+          }, {
+            default: issueTypesOptions[0].name,
+            label: "Issue Type",
+            name: "issueType",
+            type: "select",
+            options: issueTypesOptions,
+            required: true,
+          }, {
+            label: "Summary",
+            name: "summary",
+            type: "string",
+            required: true,
+          }, {
+            label: "Description",
+            name: "description",
+            type: "textarea",
+            required: false,
+          }, {
+            label: "Filename",
+            name: "filename",
+            type: "string",
+            required: false,
+          }]
+          return form
+        }
+      } catch (e) { winston.warn(`Log in fail ${JSON.stringify(e)}`) }
```

Comment:
We're leaking the access tokens in the error log here: `e.response.request.headers.Authorization`

Can we sanitize this?

[example](https://github.com/looker-open-source/actions/blob/3b8d5f117a9e3fc18a8f219b1ee1c120418bfa21/src/actions/google/common/error_utils.ts#L1) from google ads